tests/test_definitions/test_expectations_v3_api.py

```diff
@@ -125,9 +125,6 @@ def pytest_generate_tests(metafunc):  # noqa C901 - 35
                                 validator_with_data.active_batch_data,
                                 SqlAlchemyBatchData,
                             ):
-                                # Call out supported dialects
-                                if "pandas_v3_api" in only_for:
-                                    generate_test = True
                                 if "sqlalchemy" in only_for:
```

Comment:
This was a bug. The `pandas_v3_api` should have been in the if statement that was checking for `PandasBatchData`.